cupy/cusparse.py

```diff
@@ -832,6 +840,17 @@ def coo2csr(x):
         (x.data, x.col, indptr), shape=x.shape)
 
 
+def coo2csc(x):
+    handle = device.get_cusparse_handle()
+    n = x.shape[1]
+    indptr = cupy.empty(n + 1, 'i')
+    cusparse.xcoo2csr(
+        handle, x.col.data.ptr, x.nnz, n,
+        indptr.data.ptr, cusparse.CUSPARSE_INDEX_BASE_ZERO)
```

Comment:
This works for coo2csc according to the documentation of `cusparse<t>coo2csr()`:
> It can also be used to convert the array containing the uncompressed column indices (corresponding to COO format) into an array of column pointers (corresponding to CSC format).